support-frontend/assets/pages/digital-subscription-checkout/components/datePicker/datePickerHelpers.test.js

```diff
@@ -14,7 +14,7 @@ describe('datePickerHelpers', () => {
 
   class MockDate extends Date {
     constructor(param) {
-      super(param || "2020-05-14T11:01:58.135Z");
+      super(param || '2020-05-14T11:01:58.135Z');
```

Comment:
Did this change sneak in because the linter picked up the double quotes?!

Reply:
Yeah, it was failing the lint task for some reason, I'm not sure how it snuck through before!